rules/no-unique.js

```diff
@@ -4,5 +4,9 @@ const utils = require( './utils.js' );
 
 module.exports = utils.createUtilMethodRule(
 	'unique',
-	'Prefer $.uniqueSort to $.unique'
+	'Prefer $.uniqueSort to $.unique',
+	'code',
+	function ( node, fixer ) {
+		return fixer.replaceText( node.callee.property, 'uniqueSort' );
```

Comment:
isn't `node.callee.property` exactly `node` here ?

Reply:
No, `node` is the whole of `$.unique()`.

Reply:
oh right; The node is the CallExpression here.